2-mandatory/3-financial-times.js

```diff
@@ -24,14 +34,41 @@ function titleWithFewestWords(allArticleTitles) {
 */
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    const arr = allArticleTitles;
```

Comment:
I don't think there's a need to re-assign the the parametere to the new variable here.

Reply:
Thanks @berkeli yea I agree! I did it because I'm so lazy to use that long name even by using VS Code snippets :DDDD